modules/combined/tests/creep_plasticity_combined/combined_stress_prescribed.i

```diff
@@ -0,0 +1,159 @@
+#
+# 1x1x1 unit cube with time-varying pressure on top face
+#
+# The problem is a one-dimensional creep analysis.  The top face has a
+#    pressure load that is a function of time.  The creep strain can be
+#    calculated analytically.  There is no lsh plasticity.
```

Comment:
Thanks for catching this;  I'll clarify the comments to indicate that the plasticity model should not be active in this case (the yield stress is 1e30 MPa, so we shouldn't hit any plasticity during this tests).